test/azure/version-tolerant/Expected/AcceptanceTests/PagingVersionTolerant/pagingversiontolerant/operations/_operations.py

```diff
@@ -890,9 +890,7 @@ def prepare_request(next_link=None):
 
             else:
 
-                request = build_paging_duplicate_params_request(
-                    filter=filter,
-                )
```

Comment:
Even if we add the code back, the test: https://github.com/Azure/autorest.python/pull/1168/files#r818465191 still pass.

Reply:
oh yeah good point, thanks for noticing! Turns out we were always doing the right thing in version tolerant, bc for single operation in swagger paging calls, we were setting next link requests url to the `next_link`.

Still going to keep this code change because it's clearer if we never pass the query parameters to begin with (and thanks to you it also helps legacy generations with the flag). Thanks!